fgrpc/dapr.go

```diff
@@ -198,13 +257,22 @@ func (d *DaprGRPCRunnerResults) RunTest() error {
 		if t == TARGET_DAPR {
 			_, err = d.daprClient.GetState(context.Background(), d.getStateRequest)
 		}
-	} else if c == CAPABILITY_STATE {
-		if t == TARGET_DAPR {
-			_, err = d.daprClient.GetState(context.Background(), d.getStateRequest)
-		}
 	} else if c == CAPABILITY_PUBSUB {
 		if t == TARGET_DAPR {
-			_, err = d.daprClient.PublishEvent(context.Background(), d.publishEventRequest)
+			switch m {
+			case "publish":
+				_, err = d.daprClient.PublishEvent(context.Background(), d.publishEventRequest)
+			case "publish-multi":
```

Comment:
TODO: merge this with `publish`, with `numEvents` defaulting to 1.

Reply:
Thanks for @shubham1172's PR.

After review meeting with @shubham1172, we agreed with two improvements:

1. merge `publish` and `publish-multi` method, since there are almost the same except the number of publish events.
2. make numberEvents default to 1

Reply:
Thanks @skyao, updated!